threading/client/client.py

```diff
@@ -0,0 +1,396 @@
+#!/usr/bin/python3
+import socket
+import sys
+import os
+import argparse
+import signal
+import struct
+
+'''
+GLOBAL VARIABLE: FILE_LIST
+    TYPE: list()
+    BRIEF: stores list of files that exists on file server
+    USAGE: used within the RECV_FILE_LIST and GET_FILE_LIST methods
+'''
+file_list = []
+
+def socket_info():
```

Comment:
This function name does not accurately summarize the purpose of this function. Something like `parse_args` or `get_server_args` would be better.